File: backend/app/schemas/output_file_schemas.py
from pydantic import BaseModel
from datetime import datetime


class OutputFileResponseSchema(BaseModel):
    id: str
    file_name: str
    created_at: datetime
    updated_at: datetime

    class Config:
        from_attributes = True


class OutputFileCreateSchema(BaseModel):
    run_id: str
    file_name: str
    file_path: str


class OutputFileUpdateSchema(BaseModel):
    id: str

    class Config:
        from_attributes = True

File: frontend/src/types/api_types/outputFileSchemas.ts
export interface OutputFileResponse {
  id: string;
  file_name: string;
  created_at: string;
  updated_at: string;
}

export interface OutputFileCreate {
  run_id: string;
  file_name: string;
  file_path: string;
}

export interface OutputFileUpdate {
  id: string;
} 